From 943938c09e2fe44e5a20804be8ba18e72bc6d730 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Tue, 9 Apr 2013 12:02:26 +0200
Subject: [PATCH] cmake: link to pthread when wmlib is used

wmlib explicitly uses pthread, so make sure to link to it without relying on something else
---
 CMakeLists.txt |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 59aeaa5..1ab8721 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,6 +63,9 @@ target_link_libraries(kcompactdisc ${KDE4_KDECORE_LIBS} ${KDE4_PHONON_LIBS} ${KD
 if (HAVE_LIBASOUND2)
     target_link_libraries(kcompactdisc ${ASOUND_LIBRARY})
 endif (HAVE_LIBASOUND2)
+if (USE_WMLIB)
+    target_link_libraries(kcompactdisc ${CMAKE_THREAD_LIBS_INIT})
+endif (USE_WMLIB)
 
 set_target_properties(kcompactdisc PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION} )
 
-- 
1.7.10.4

